Libplanet/Store/LiteDBStore.cs

```diff
@@ -572,6 +461,13 @@ private string BlockStateFileId(HashDigest<SHA256> blockHash)
             return $"state/{blockHash}";
         }
 
+        private string StateRefId(string @namespace)
+        {
+            string ns =
+                ByteUtil.Hex(namespaceHasher.ComputeHash(Encoding.UTF8.GetBytes(@namespace)));
```

Comment:
Is there a reason to use the hash of namespace instead of the namespace?

Reply:
Because [LiteDB's collections names allow only limited set of characters][1], and on `IStore`-level namespaces are just free text strings.  IMHO, namespaces should be rather `Guid` than `string`.

[1]: https://github.com/mbdavid/LiteDB/wiki/Collections